pkg/dashboards/dashboard.go

```diff
@@ -97,7 +97,10 @@ const getDashboardEntityQuery = `query ($guid: EntityGuid!) {
             layout { column height row width }
             title
             visualization { id }
-            id
+						linkedEntities {
+							__typename
+							guid
```

Comment:
Should we include more fields here?

Reply:
Need to keep `id`, which appears to have been dropped.  Can you convert to spaces for consistency within the string?

`GUID` is the only required input field for adding the link to a widget, `name` and `accountId` would be useful for validation / human consumption.  The data set would be rather small on fetch, so adding more fields should have minimal impact if any.  Maybe something like this:

```
            linkedEntities {
              __typename
              guid
              name
              accountId
              tags {
                key
                values
              }
              ... on DashboardEntityOutline {
                dashboardParentGuid
              }
            }
```

Dashboard entities are the only supported type today, might be helpful to have the parent GUID, might never be used.

Reply:
Oh crap, I didn't even realize I removed that. Good catch.

Reply:
Thanks for the suggestions. Updated